examples/convert.rs

```diff
@@ -106,8 +106,7 @@ fn main() {
             let input = fs::read_to_string(&args[1]).unwrap();
             naga::front::glsl::parse_str(
                 &input,
-                "main",
-                naga::ShaderStage::Vertex,
+                vec![("main".to_string(), naga::ShaderStage::Vertex)],
```

Comment:
time to introduce a `struct Options`!

Reply:
Ok, you thinking `Vec<StageOptions>` or `GlslParserOptions` as general options struct for glsl parser?

Reply:
Just a single `struct Options` that's passed into the frontend by reference, similar to how SPV-out and MSL-out are structured.